src/cclib/io/ccio.py

```diff
@@ -363,6 +364,8 @@ def _determine_output_format(outputtype, outputdest):
             outputclass = cmlwriter.CML
         elif outputtype.lower() == 'xyz':
             outputclass = xyzwriter.XYZ
+        elif outputtype.lower() == 'molden':
```

Comment:
https://github.com/cclib/cclib/pull/385/commits/96d30ccd04b43b73e671a7c468711c8f3b22b6f2 Moved the mappings to a dictionary.